lib/chef/resource.rb

```diff
@@ -1382,9 +1385,16 @@ def self.action(action, &recipe_block)
     #   created resource with its identity values filled in.
     #
     def self.load_current_value(&load_block)
+      include LoadCurrentValueDSL
```

Comment:
Because it doesn't make sense (it's an appendage) unless you are actually using `load_current_value`.  Happy to add it, but it's just a method that doesn't do anything for you unless you have load_current_value.